addons/mrp/models/mrp_bom.py

```diff
@@ -176,6 +176,10 @@ def name_create(self, name):
             raise UserError(_("You cannot create a new Bill of Material from here."))
         return super(MrpBom, self).name_create(name)
 
+    def toggle_active(self):
+        self.with_context({'active_test': False}).operation_ids.toggle_active()
+        return super(MrpBom, self).toggle_active()
```

Comment:
```suggestion
        return super().toggle_active()
```
Filling the super is the old way dating from python 2